multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -59,6 +59,29 @@ coreos:
         [Install]
         RequiredBy=kubelet.service
 
+    {{ if .Experimental.NodeDrainer.Enabled }}
+    - name: kube-node-drainer.service
+      enable: true
+      content: |
+        [Unit]
+        Description=drain this k8s node to make running pods time to gracefully shut down before stopping kubelet
```

Comment:
Looks like systemd shutdown down services in the reverse of the boot order, so `After=kubelet.service` should mean this should finish shut down before the kubelet is.